frontend/public/texts/landing_page_texts.tsx

```diff
@@ -145,5 +145,31 @@ export default function getLandingPageTexts({ classes, isNarrowScreen }) {
       en: "Learn from",
       de: "Lerne von",
     },
+    return_to_hubs: {
+      en: "Return to hubs",
+      de: "Zurück zu den Knotenpunkten",
+    },
+    find_suitable_climate_protection_commitment: {
+      en: "Find your suitable climate protection commitment in",
+      de: "Finde dein passendes Klimaschutz Engagement in",
+    },
+    climateHub: {
+      en: "ClimateHub",
+      de: "ClimateHub",
+    },
+    find_fellow_campaigners_for_climate_protection_idea: {
+      en:
+        "Find fellow campaigners for your climate protection idea, share your climate project with others. Register now and make a difference with the right commitment. Do you need support? ",
+      de:
+        "Finde Mitstreiter:innen für deine Klimaschutz Idee, teile dein Klima Projekt mit anderen. Jetzt anmelden und mit dem richtigen Engagement einen Unterschied machen. Brauchst du Unterstützung? ",
+    },
```

Comment:
Here we'll still need to think of something more SEO optimized, but I'd suggest this for now

```suggestion
    find_fellow_campaigners_for_climate_protection_idea: {
      en:
        "Find projects and initiatives from Erlangen to get involved and make a difference. The ClimateHub is your starting point to find your group to make an impact.
      de:
        "Finde Mitstreiter:innen, um in Erlangen gemeinsam für den Klimaschutz anzupacken und etwas für eine nachhaltige Zukunft zu bewirken."
    },
```